controller/iteration_test.go

```diff
@@ -347,7 +348,7 @@ func (rest *TestIterationREST) TestFailUpdateIterationNotFound() {
 
 func (rest *TestIterationREST) TestFailUpdateIterationUnauthorized() {
 	// given
-	_, _, _, itr := createSpaceAndRootAreaAndIterations(rest.T(), rest.db)
+	_, _, _, _, itr := createSpaceAndRootAreaAndIterations(rest.T(), rest.db)
```

Comment:
You return this new area object but you don't check it in any of the tests. Why ?